skbio/io/format/_base.py

```diff
@@ -227,13 +229,18 @@ def _line_generator(fh, skip_blanks=False, strip=True):
 def _too_many_blanks(fh, max_blanks):
     count = 0
     too_many = False
+    consumed = []
     for line in _line_generator(fh, skip_blanks=False):
+        consumed += [line]
```

Comment:
This can be replaced with:

```python
consumed.append(line)
```

Reply:
Is this more efficient than just doing +=? I'm using += in other places as sometimes I need to add multiple lines, and wanted to keep it consistent.

Reply:
`append` is just a "pythonic" way :) I don't know if it's more efficient or not. PS: if  you want to add multiple elements, you can do `consumed.extend([line1, line2, line3...])`.